test/flows/student_finance_calculator_flow_test.rb

```diff
@@ -0,0 +1,519 @@
+    end
+
+    should "render text if student has a disability" do
+      add_responses do_any_of_the_following_apply_uk_full_time_students_only?: "has-disability"
+
+      assert_rendered_outcome text: "Disabled Students’ Allowances"
+    end
+
+    should "render text if student has low income" do
+      add_responses do_any_of_the_following_apply_uk_full_time_students_only?: "low-income"
+
+      assert_rendered_outcome text: "University and college hardship funds"
+    end
+
+    should "render text if student is eligible for teacher training funding" do
+      assert_rendered_outcome text: "Funding for teacher training"
+    end
+
+    should "render text if student is eligible for social work funding" do
+      add_responses what_course_are_you_studying?: "social-work"
+
+      assert_rendered_outcome text: "Social Work Bursary"
+    end
+
+    should "render the full time student text within the extra help partial" do
+      assert_rendered_outcome text: "You might be able to get help with the costs of travel for study or work placements"
+    end
+  end
+
+  context "outcome: outcome_uk_part_time_students" do
+    setup do
+      testing_node :outcome_uk_part_time_students
+      add_responses when_does_your_course_start?: "2020-2021",
+                    what_type_of_student_are_you?: "uk-part-time",
+                    how_much_are_your_tuition_fees_per_year?: "6935",
+                    where_will_you_live_while_studying?: "at-home",
+                    whats_your_household_income?: "50,000",
+                    how_many_credits_will_you_study?: "10",
+                    how_many_credits_does_a_full_time_student_study?: "11",
+                    do_any_of_the_following_apply_all_uk_students?: "has-disability",
+                    what_course_are_you_studying?: "dental-medical-healthcare"
+    end
+
+    should "render text if the student is studying a medical course" do
+      assert_rendered_outcome text: "To qualify for a Maintenance Loan you must be studying a DipHE in dental hygiene and dental therapy"
+    end
+
+    should "render text if the student is studying a teaching course" do
+      add_responses what_course_are_you_studying?: "teacher-training"
+
+      assert_rendered_outcome text: "To qualify for a Maintenance Loan you must be studying an Initial Teacher Training course"
+    end
+
+    should "render text if the student is not studying a maintenance loan qualified course" do
+      add_responses what_course_are_you_studying?: "none-of-the-above"
+
+      assert_rendered_outcome text: "To qualify for a Maintenance Loan, you must be studying one of the following courses"
+    end
+
+    should "not render text if the student is not eligible for a maintenance loan" do
+      add_responses how_many_credits_will_you_study?: "2",
+                    how_many_credits_does_a_full_time_student_study?: "10"
+
+      assert_no_match "How your Maintenance Loan is calculated", @test_flow.outcome_text
```

Comment:
I'd add the counter test to this as well, so add a test that has `assert_rendered_outcome text: "how your maintenance....